make_benchmark/zenodo_upload.py

```diff
@@ -0,0 +1,141 @@
+"""
+Zenodo Uploader: Uploads the datasets to the Geo-Bench community on Zenodo.
+
+By: Alexandre Drouin (ServiceNow Research)
+Date: June 2, 2023
+
+"""
+import json
+import os
+import requests
+
+from path import Path
+
+
+ACCESS_TOKEN = os.environ["ZENODO_TOKEN"]
+
+VERSION = "v0.9.1"
+
+AUTHORS = [
+    {"name": "Lacoste, Alexandre", "affiliation": "ServiceNow Research"},
+    {"name": "Lehmann, Niels", "affiliation": "University of Amsterdam"},
+    {"name": "Rodriguez, Pau", "affiliation": "ServiceNow Research"},
+    {"name": "David Sherwin, Evan", "affiliation": "Stanford University"},
+    {"name": "Kerner, Hannah", "affiliation": "University of Maryland"},
+    {"name": "Lütjens, Björn", "affiliation": "Massachusetts Institute of Technology"},
+    {"name": "Irvin, Jeremy", "affiliation": "Stanford University"},
+    {"name": "Dao, David", "affiliation": "ETH Zurich"},
+    {"name": "Alemohammad, Hamed", "affiliation": "Clark University"},
+    {"name": "Drouin, Alexandre", "affiliation": "ServiceNow Research, Mila-Quebec"},
+    {"name": "Gunturkun, Mehmet", "affiliation": "ServiceNow Research"},
+    {"name": "Huang, Gabriel", "affiliation": "ServiceNow Research, University of Montreal"},
+    {"name": "Vazquez, David", "affiliation": "ServiceNow Research"},
+    {"name": "Newman, Dava", "affiliation": "Massachusetts Institute of Technology"},
+    {"name": "Bengio, Yoshua", "affiliation": "Mila-Quebec, University of Montreal"},
+    {"name": "Ermon, Stefano", "affiliation": "Stanford University"},
+    {"name": "Xiang Zhu, Xiao", "affiliation": "Technical University of Munich"},
+]
+
+DATASETS = [
+    # Classification datasets
+    ("m-so2sat", f"classification_{VERSION}/so2sat", "cc-by-4.0"),
+    ("m-bigearthnet", f"classification_{VERSION}/bigearthnet", "cdla-permissive-1.0"),
+    ("m-brick-kiln", f"classification_{VERSION}/brick_kiln_v1.0", "cc-by-sa-4.0"),
+    ("m-forestnet", f"classification_{VERSION}/forestnet_v1.0", "cc-by-4.0"),
+    ("m-eurosat", f"classification_{VERSION}/eurosat", "cc-by-4.0"),
+    ("m-pv4ger", f"classification_{VERSION}/pv4ger_classification", "mit"),
+    # Segmentation datasets
+    (
+        "m-chesapeake-lancover",
+        f"segmentation_{VERSION}/cvpr_chesapeake_landcover",
+        "cdla-permissive-1.0",
+    ),
+    ("m-NeonTree", f"segmentation_{VERSION}/NeonTree_segmentation", "cc-by-4.0"),
+    ("m-nz-cattle", f"segmentation_{VERSION}/nz_cattle_segmentation", "cc-by-4.0"),
+    ("m-pv4ger-seg", f"segmentation_{VERSION}/pv4ger_segmentation", "mit"),
+    ("m-cashew-plantation", f"segmentation_{VERSION}/smallholder_cashew", "cc-by-4.0"),
+    ("m-SA-crop-type", f"segmentation_{VERSION}/southAfricaCropType", "cc-by-4.0")
+    # Test dataset
+    ("test-dataset", f"classification_test_{VERSION}/test_dataset", "mit"),
```

Comment:
I did this on purpose in case you want to change something in future versions and need to keep the past version of classification_test for a unit test.